tock/hours/models.py

```diff
@@ -314,6 +316,10 @@ class TimecardNoteManager(models.Manager):
     def enabled(self):
         return super(TimecardNoteManager, self).get_queryset().filter(enabled=True)
 
+    def active(self):
+        now = timezone.now()
+        return super(TimecardNoteManager, self).get_queryset().filter(Q(enabled=True) | Q(display_period_start__lte=now, display_period_end__gt=now))
```

Comment:
Wouldn't this blow up if start and end were left blank?

Reply:
Based on my testing and understanding, we should be okay here. The field lookups (`lt`, `gt`, etc) resolve to false in cases where the value is `null`